inductiva/utils/visualization.py

```diff
@@ -26,6 +32,65 @@
 }
 
 
+def create_movie_from_widget(view,
```

Comment:
Should this function be called from `render_interactive` or from somewhere else?

Reply:
I don't want to put it in the render_interactive because it messes up the visualization, because whenever you do render_image it updates it. So I would leave it like this and if the user wants to save the video he calls this directly and when we have a better solution in the backend we can have the functionality implemented in the class.

Reply:
Sounds good, thanks for explaining the rationale!